components/PageHeader/pageheader.scss

```diff
@@ -0,0 +1,36 @@
+@import '../../styles/responsive.scss';
+
+.pageheader-container {
+
+  padding: 0;
+  margin-top: 1rem;
+  margin-bottom: 2rem;
+
+  @include device(lg) {
+    display: flex;
+    justify-content: space-between;
+    align-items: start;
```

Comment:
L12: I saw various errors in the stack because we need to switch `start` to `flex-start`.